packages/cli/src/cmds/preview/constants.ts

```diff
@@ -107,7 +107,9 @@ export const doctorResult = {
   NodeNotSupported: `Node.js (@CurrentVersion) is not in the supported version list (@SupportedVersions).`,
```

Comment:
No BI for Node.js Settings. 
BI of Func-core-tools Settings and Dotnet Settings have been removed since **2022/02/16**.
I will add telemetry for skipping node checker and func-core-tools checker in later PR.